common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+		gold_cost = 250
+		build_time = 180
+		ai_creation_factor = 95
+		
+		tech_growth_modifier = 0.125
+		military_techpoints = 0.01
+		tradevalue = 15
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game	
+	}
+	
+	##Nigth Elf Group
+	tp_resting_lounge = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = night_elf_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 225
+		build_time = 180
+		ai_creation_factor = 95
+		
+		monthly_character_piety = 0.03
+		garrison_size = 0.5
+		tech_growth_modifier = 0.125
+		cultural_techpoints = 0.02
+		tradevalue = 10
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+		
+	##Humans, High elves, Blood elves and Draenei
+	tp_jewelcrafters = {
+		potential = {
+			in_trade_route_trigger = yes 
+			FROM = { 			
+				OR = {
+					culture_group = high_elf_group
+					culture_group = human_group
+					culture_group = draenei_group
+				}	
+			}		
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 475
+		build_time = 360
+		ai_creation_factor = 95
+		
+		economic_techpoints = 0.01
+		tradevalue = 30
+		trade_route_wealth = 4		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+
+	##Tauren/Taunka
+	tp_kodo_breeders = {
+		potential = {
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = tauren_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Tuskar Group
+	tp_turtle_pools = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = tuskarr_group }		
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Orc Group
+	tp_wolf_bestiary = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = orc_group }			
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 275
+		build_time = 180
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Harpy Group
+	tp_guide_eagle_nests = {
+		potential = { 
+			in_trade_route_trigger = yes 
+			FROM = { culture_group = harpy_group }
+		}
+		desc = tp_empty
+		trigger = { TECH_TRADE_PRACTICES = 3 }
+		gold_cost = 300
+		build_time = 270
+		ai_creation_factor = 95
+		
+		retinuesize = 20
+		tech_growth_modifier = 0.2		

```

Comment:
[The Spreadsheet](https://docs.google.com/spreadsheets/d/1KLHQRAANjXuKB1PYqk-aJu_60DV_KCLXiyZjJV7mwNU/edit?usp=sharing) says it should be...
```suggestion
		tech_growth_modifier = 0.125
```